python/cudf/cudf/tests/test_struct.py

```diff
@@ -194,12 +192,6 @@ def test_struct_scalar_host_construction_no_dtype_inference(data, dtype):
     # is empty.
     slr = cudf.Scalar(data, dtype=dtype)
     assert slr.value == data
-    assert list(slr.device_value.value.values()) == list(data.values())
-
-
-def test_struct_scalar_null():
```

Comment:
Are we losing this test permanently, or will we get is back?

Reply:
Thanks, I found a way to add this test back in a different form